src/menu/cheat_load.c

```diff
@@ -0,0 +1,197 @@
+#include <string.h>
+
+#include <libdragon.h>
+
+#include <stdio.h>
+#include <sys/stat.h>
+#include "views/views.h"
+#include "cheat_load.h"
+#include "../utils/fs.h"
+
+/**
```

Comment:
Move this to the top of the document. Bonus points is to make it doxygen compliant 😉